sass/pages/_reset-password.scss

```diff
@@ -0,0 +1,100 @@
+.reset-password__container {
+  @include row {
+    justify-content: center;
+    margin-top: 61px;
+  }
+
+  .reset-password {
+    @include column;
+    .reset-password__footer {
+      @include row {
+        justify-content: center;
+        align-items: center;
+      }
+      height: 65px;
+      flex-grow: 1;
+      background: $white-shade;
+      p {
+        font-size: 14px;
```

Comment:
you can create a variable for font-size so that everyone will keep the same font-size.